deploy/createTypesPackages.js

```diff
@@ -54,6 +54,17 @@ export const packages = [
       { from: "../generated/ts5.5/sharedworker.iterable.generated.d.ts", to: "ts5.5/iterable.d.ts", autoImport: true, group: "ts5.5" },
     ],
   },
+  {
+    name: "@types/webworker",
+    description: "Types for Web Worker",
```

Comment:
```suggestion
    description: "Types for the global scope of Web Workers",
```

Note the others above.